static/app/views/performance/transactionSummary/transactionSpans/content.tsx

```diff
@@ -64,11 +70,19 @@ function SpansContent(props: Props) {
     };
   }
 
+  const spanOp = decodeScalar(location.query.spanOp);
```

Comment:
Not for this PR, but as an aside, looking at how many of these we end up doing, I feel like we should have a `useLocation` and then have `useQueryField('QueryField.SPAN_OP')` eventually, with enums to ensure we don't have typos selecting from the query 🤔

Reply:
That makes sense to me, we can follow up with this later.